### PR TITLE
[DO NOT MERGE] Create Job to test releases branches in Eclipse Che

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -570,6 +570,11 @@ cherry_pick_unapproved:
 config_updater:
   gzip: false
   maps:
+    ci-operator/config/**/*-7*.yaml:
+      clusters:
+        app.ci:
+        - ci
+      name: ci-operator-misc-configs
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:577]: An analogous one is needed for `ci-operator/jobs` path

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -816,6 +821,11 @@ config_updater:
         - ci
       gzip: true
       name: job-config-misc
+    ci-operator/jobs/**/*-7*.yaml:
+      clusters:
+        app.ci:
+        - ci
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:827]: Add a item for `api.ci` just like the other ones for jobs have.

File: core-services/prow/02_config/_plugins.yaml [hunk 3]
```diff
@@ -570,6 +570,11 @@ cherry_pick_unapproved:
 config_updater:
   gzip: false
   maps:
+    ci-operator/config/**/*-7*.yaml:
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:573]: That's covering way too much, please limit the pattern up to the appropriate org or even better, a repo

File: core-services/prow/02_config/_plugins.yaml [hunk 4]
```diff
@@ -816,6 +821,11 @@ config_updater:
         - ci
       gzip: true
       name: job-config-misc
+    ci-operator/jobs/**/*-7*.yaml:
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:824]: That's covering way too much, please limit the pattern up to the appropriate org or even better, a repo

**Review by petr-muller — APPROVED** [review]
/hold

Merge at will

**Review by tolusha — APPROVED** [review]

**Comment by flacatus** [comment]
/ test ?

**Pull request merged**